tools/harness-automation/autothreadharness/runner.py

```diff
@@ -143,15 +143,17 @@ def addError(self, test, err):
         super(SimpleTestResult, self).addError(test, err)
         self.add_result(test, None, str(err[1]))
 
-def list_devices(names=None, continue_from=None, **kwargs):
+def list_devices(continue_from=None, **kwargs):
     """List devices in settings file and print versions"""
 
     if continue_from:
         continue_from = settings.GOLDEN_DEVICES.index(continue_from)
     else:
         continue_from = 0
 
-    for port in names or settings.GOLDEN_DEVICES[continue_from:]:
```

Comment:
`names` argument allows for listing certain devices, for example
```
./start.sh -l COM1 COM2
```

Better to select `OpenThread` device names from `settings.GOLDEN_DEVICES` if no `names` are given.